### ISSUE TITLE
New serial commands for arduino

### ISSUE THREAD

**Issue by silvapacita** [JBoye/HACKTANE#5]
When you can, please add the commands for lights on/off/alarm to the serial (90,91,92), so that we can add relays to our arduinos (for those who don't have Hue).

cheers

**Comment by JBoye** [comment]
I was thinking about adding these commands for the interface: What do you think?

 * 90 - Lights off
 * 91 - Lights on
 * 92 - Warning Lights on

 * 95 - Alarm clock off
 * 96 - Alarm clock on

* 100 -Game started (loaded)
* 101 - Timer started
* 102 - Bomb Defused
* 103 - Bomb Exploded - Time
* 104 - Bomb Exploded - Strike

**Comment by silvapacita** [comment]
yes, those commands would enhance the arduino combo that we could do with relays and output pins. For now, i can't remember any other command that you could add.

**Comment by JBoye** [comment]
Cool. 
There is a list of sound effects in the [modframework](https://github.com/keeptalkinggame/ktanemodframework/blob/master/KMFramework/src/KMSoundOverride.cs ) for inspiration.

Maybe there should be a command for "CorrectChime", whenever a module is solved. NeedyActivated/NeedyWarning could also be useful.

**Comment by silvapacita** [comment]
...the more, the better :D

**Comment by silvapacita** [comment]
Don't know if it's possible to add this in the mod, but you could add input to the game from the arduino usb (for example, to turn the alarm off). The digital pin 1 and 0 are actually tx and rx serial connection, so i can add a button to the arduino code, and send a signal trough usb, that would be read by the Hacktane.

**Comment by JBoye** [comment]
It’s not possible at the moment, but I hope the Ktane devs will add it at some point. 
I’m close to releasing a new version of Hacktane though, completely rewritten from the ground up. I’ve been mostly focusing on the Hue for now, but there will be plenty of improvements for the arduino as well

**Comment by silvapacita** [comment]
![img_20171112_181310](https://user-images.githubusercontent.com/32032474/32701935-5f110346-c7d6-11e7-8d5d-4fc6675397de.jpg)
Just to let know that the Hacktane pcb shield for Arduino Uno is almost done ;)
will post photos when complete and files for creating one

**Comment by RogierQ** [comment]
> I was thinking about adding these commands for the interface: What do you think?
> 
> * 90 - Lights off
> * 91 - Lights on
> * 92 - Warning Lights on
> * 95 - Alarm clock off
> * 96 - Alarm clock on
> * 100 -Game started (loaded)
> * 101 - Timer started
> * 102 - Bomb Defused
> * 103 - Bomb Exploded - Time
> * 104 - Bomb Exploded - Strike

Is this added to the mod?
Cannot get this as result
